packages/tools/src/utilities/math/polyline/pointCanProjectOnLine.ts

```diff
@@ -0,0 +1,52 @@
+import type { Types } from '@cornerstonejs/core';
+import { vec2 } from 'gl-matrix';
+
+/**
+ * Returns `true` if the point `p` can project onto point (`p1`, `p2`), and if
+ * this projected point is less than `proximity` units away.
+ */
+const pointCanProjectOnLine = (
```

Comment:
The function name does not match its output, I expected a boolean output, but it is number | false